app/views/content/funding-and-support/promos/_get-adviser-funding-promo.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Get an adviser", classes: %w[tta-background]) do %>
```

Comment:
@gazcarr the image on this promo is cropped pretty badly in mobile view. Is there anything we can do to fix it?

Reply:
Could you point me in the direction of the page you're referring to? Can't quite work out where it is

Reply:
It's the 'Get an adviser' promo and it's on a few of the pages in the funding and support section. E.g /funding-and-support/scholarships-and-bursaries. To be fair it's not much different to what's live eg in the train to be a teacher section. But when I view on mobile setting I see this 
![image](https://user-images.githubusercontent.com/107919390/177136496-7e4ab8c4-361d-4af1-a626-85ff456c34e4.png)

Reply:
SO I am ok with this, on the basis that its a background image, but I know in the past @gemmadallmandfe has preferred content to be more central in the view. That  being said, this is one for @ethax-ross to adjust if we do decide